src/main/java/com/onlinebookstore/store/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,27 @@
 package com.onlinebookstore.store.dto;
 
+import com.onlinebookstore.store.validation.Description;
+import com.onlinebookstore.store.validation.Isbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @NotBlank
+    @Isbn
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotBlank
+    @Description
```

Comment:
```suggestion
    @Size
```

don't need to create custom annotation to validate size